src/workerd/api/http-standard-test.js

```diff
@@ -0,0 +1,27 @@
+import { strictEqual } from 'node:assert';
+
+export default {
+  async fetch(req) {
+    if (req.url.endsWith('/')) {
+      return new Response(null, {
+        status: 302,
+        headers: {
+          location: ' /\t2  '
```

Comment:
Am I reading this right? The standards-based parser strips tabulator characters?

Reply:
Yep. tabs and newlines that are interspersed through the input are removed, as is any leading/trailing whitespace. Spaces within the input are handled by converting to `%20`, but they can also be represented at `+`